.github/renovate.json5

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": [
+    "config:recommended"
+  ],
+  "mode": "full",
```

Comment:
Why?

Reply:
I think it would be beneficial here since there are three versioned dependencies that I think no one is tracking:

- https://github.com/pikaviestin/website-www.pikaviestin.fi/blob/986964e7fb2082e2f52b99a3da916897ac7e1f01/.github/workflows/main.yaml#L19
- https://github.com/pikaviestin/website-www.pikaviestin.fi/blob/986964e7fb2082e2f52b99a3da916897ac7e1f01/.github/workflows/main.yaml#L21
- https://github.com/pikaviestin/website-www.pikaviestin.fi/blob/986964e7fb2082e2f52b99a3da916897ac7e1f01/.github/workflows/main.yaml#L38

Additionally there are the Node dependencies that are yet to be versioned:

- `md-to-pdf`
- #19 would add `prettier`, `prettier-plugin-sh` (which regardless of the name also affects git configs and other configs) and `packageManager`

-----

The full mode means renovate would open dependency dashboard issue as opposed to default silent mode where it would just open pull requests without warning.

I should also link to https://github.com/apps/forking-renovate which is what I am using and which won't write to the repository directly.

Reply:
We are already using the non-forking renovate in other repos.